amundsen_application/static/js/components/common/BadgeList/index.tsx

```diff
@@ -23,6 +26,14 @@ const BadgeList: React.FC<BadgeListProps> = ({ badges }: BadgeListProps) => {
         // metadata badges with name and category
         else if (badge.badge_name) {
           badgeConfig = getBadgeConfig(badge.badge_name);
+          if (badge.category === COLUMN_BADGE_CATEGORY) {
+            return (
+              <Flag
+                text={badgeConfig.displayName}
+                labelStyle={badgeConfig.style}
+              />
+            );
+          }
         }
         return (
           <ClickableBadge
```

Comment:
Thinking twice about this, maybe it would make sense to separate the concept of a Badge (interactive) vs a Flag, so creating a FlagList to handle all this case.

WDYT?

Reply:
I feel like all of the functionality is so similar (and domain wise we refer to column level and table badges as exactly the same thing), that it might be worth keeping the list the same and them differentiating between clickable and not clickable badges in this point, I do believe that we could define those components differently in this file. Maybe we could do a refactor where we pass the on click action for triggering search as an optional parameter, and if it is present then we proceed to make the badge clickable since if you look in the ClickableBadge component you'll notice there is a Flag in there.

Reply:
That sounds good too!

Reply:
Wait now looking at it I don't believe this code would work with multiple badges in one badgeList??

Reply:
Actually no because it is within that map function I forgot about